docs/SUMMARY.md

```diff
@@ -132,6 +132,7 @@
 * [Pipeline Plugins](user-guide/plugins/README.md)
   * [Create Your Plugin](user-guide/plugins/create-plugin.md)
   * [Our Plugins](user-guide/plugins/plugin-list.md)
+    * [BitBucket Runner Trigger](user-guide/plugins/bitbucket-runner-trigger.md)
```

Comment:
Replace `BitBucket` with `Bitbucket`
We need to get the capitalization fixed by the devs too.

Reply:
fixed